.circleci/config.yml

```diff
@@ -46,7 +48,12 @@ references:
         fi
         epmd -daemon
         make python-env
-        make python-release-test TARBALL=${PACKAGES_DIR:?}/epoch-$(cat VERSION)-${PKG_SUFFIX}.tar.gz
+        mkdir ${PACKAGE_TESTS_DIR:?}
```

Comment:
@dincho I suspect a potential failure on this mkdir when running both linux_package and osx_package though based on my tests I am reasonably confident there is no conflict. I aim at definitely proving it with [this build that is the same branch as this plus the osx_package job enabled](https://circleci.com/gh/lucafavatella/epoch/273).

Reply:
Proved.